utils/db.ts

```diff
@@ -308,6 +306,7 @@ export async function createUser(user: User) {
     .set(usersKey, user)
     .set(usersByLoginKey, user)
     .set(usersBySessionKey, user)
+    .sum(analyticsKey("users_count"), 1n)
```

Comment:
Hm... It might be better to define a `const usersCount` at the start of the function with the others without using `analyticsKey()`. It gives a more immediate idea of what's going on, IMO. At most, maybe a `toDate()` that returns `date.toISOString().split("T")[0]` might be useful. Ditto for other similar scenarios. WDYT?

Reply:
I.e. check out `formatDate()` in #289.

Reply:
Makes sense. Will adjust that!